cache.go

```diff
@@ -166,12 +166,12 @@ type Item[V any] struct {
 // NewCache returns a new Cache instance and any configuration errors, if any.
 func NewCache[K Key, V any](config *Config[K, V]) (*Cache[K, V], error) {
 	switch {
-	case config.NumCounters == 0:
```

Comment:
Could you update this such that we do not change existing errors? This would mean keeping the existing code as is and introducing new cases and new errors.

Reply:
I'm very sorry, I may have overlooked this case, and I will revise my PR.